Cargo.lock

```diff
@@ -6456,13 +6456,27 @@ dependencies = [
 
 [[package]]
 name = "sgx_build_helper"
-version = "1.1.5"
-source = "git+https://github.com/apache/incubator-teaclave-sgx-sdk?tag=v1.1.5#d2d339cbb005f676bb700059bd51dc689c025f6b"
+version = "1.1.6"
+source = "git+https://github.com/apache/incubator-teaclave-sgx-sdk?branch=v1.1.6-testing#9c1bbd52f188f600a212b57c916124245da1b7fd"
 
 [[package]]
 name = "sgx_crypto_helper"
 version = "1.1.5"
 source = "git+https://github.com/apache/incubator-teaclave-sgx-sdk?tag=v1.1.5#d2d339cbb005f676bb700059bd51dc689c025f6b"
```

Comment:
I think we should investigate where this dependency comes from. I am a bit wary of using an old one.

Reply:
It seems to have come from `core/tls-websocket-server/Cargo.toml`.
Fixed this in bd750ad7d3c4a6d971136b12e0c95765f2ed37eb.
I will check the patches section of the `cargo workspace` though, It should have been overridden by that. :confused:

Reply:
Nope, patches don't work across different versions.

Thanks for fixing it! ❤️